envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/listeners/filters/RBACFilterFactory.kt

```diff
@@ -29,9 +33,10 @@ class RBACFilterFactory(
     }
 
     private val anyPrincipalName = "_ANY_"
+    private val exactIpMask = 32
```

Comment:
I think we should allow to modify this

Reply:
Do you think this should be per service configuration or global EC?